app/components/add-proposal/component.js

```diff
@@ -13,47 +14,51 @@ export default Component.extend({
   kredits: service(),
 
   proposal: null,
+  contributors: null,
   inProgress: false,
 
   isValidRecipient: computed('proposal.recipientAddress', function() {
-    return this.get('kredits.web3Instance').isAddress(this.get('proposal.recipientAddress'));
+    return this.get('kredits.web3').isAddress(this.get('proposal.recipientAddress'));
```

Comment:
Not "now", but in general, yes. I don't know why it's not in the add-contributor component.